Symfony/CS/Tests/Fixer/Symfony/NoEmptyLinesAfterPhpdocsFixerTest.php

```diff
@@ -212,4 +212,26 @@ public function testFixesWindowsStyle()
 
         $this->makeTest($expected, $input);
     }
+
+    // empty line between typehinting docs and return statement should be preserved
+    public function testInlineTypehintingDocsBeforeReturn()
+    {
+        $expected = <<<'EOF'
+<?php
+
+function parseTag($tag)
+{
+    $tagClass = get_class($tag);
+
+    if ('phpDocumentor\Reflection\DocBlock\Tag\VarTag' === $tagClass) {
+        /** @var DocBlock\Tag\VarTag $tag */
```

Comment:
Fixer is supposed to remove empty line between docblock and documented content.
Here, it is not documented content.
Other way would be to run after phpdoc_to_comment, which may be a good way as well. Need to think more about it.